docs/tsc/meetings/2019-04-15.md

```diff
@@ -0,0 +1,60 @@
+Apr 11, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Carl Rand
+
+Attendees:
+  * [ ] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [ ] Thanh Ha - Academy Software Foundation / Linux Foundation
```

Comment:
As Thanh is not anymore working for the Linux Foundation, his name could be removed from the list.